manifests/manifest-base.yml

```diff
@@ -2,8 +2,10 @@
 buildpack: go_buildpack
 memory: 256M
 services:
-  - deck-ups
+- deck-ups
+- deck-redis
```

Comment:
I'm not sure if renaming a service will affect the way the app parses the VCAP_SERVICES, but you can run this @msecret and submit a PR which updates the VCAP lookup.

https://docs.cloudfoundry.org/devguide/services/managing-services.html#rename_service